COMPONENT_GUIDE.md

```diff
@@ -252,3 +252,39 @@ Text
 
 </Caution>
 ```
+
+## Related Resources
```

Comment:
I'm a bit hesitant to include this section in the `COMPONENT_GUIDE.md` document, only because this document documents components available _in the markdown_ itself. This is directly referencing _frontmatter_ specific information, so its not something you'd use or reference in the markdown directly.

That being said, maybe its ok that we do this, but maybe we should consider whether we should include other useful frontmatter specific details. It kinda feels caught in between the style guide doc and this one otherwise. This information is super useful, so I'm not advocating against not including any of it, just figuring out where best to put it 😄 

Thoughts?

Reply:
Well it's how the component works, and the style guide references how the frontmatter works. I think it's OK.